tox.ini

```diff
@@ -10,49 +10,54 @@ isolated_build = True
 [testenv]
 commands = python -m pip install --upgrade pip
 
-[testenv:dev]
-description = development environment
-usedevelop = True
-deps =
-    black
-    pip-tools
-    flake8
-    flakehell
-    pytest
-    pytest-datafiles
-commands =
-    python -m pip install --upgrade pip
-    pip-compile requirements/base.in
-    pip-compile requirements/dev.in
-    pip-compile requirements/docs.in
-    pip-compile requirements/tests.in
-    pip install -q -e . -r requirements/dev.txt
 
 [testenv:tests]
 deps =
-    -rrequirements/base.txt
+    -rrequirements.txt
     pytest
     pytest-datafiles
 commands = python -m pytest --basetemp={envtmpdir} {posargs}
 
+
 [testenv:linting]
-skip_install = true
 deps =
-    -rrequirements/base.txt
+    -rrequirements.txt
     pylint
 commands = pylint src/bo4e
 
+
 [testenv:coverage]
 deps =
     {[testenv:tests]deps}
     coverage
 commands =
     coverage run -m pytest --basetemp={envtmpdir} {posargs}
+    coverage html --omit .tox/*,unittests/*
     coverage report --fail-under 80 --omit .tox/*,unittests/*
 
 
+[testenv:dev]
+usedevelop = True
+commands =
+    python -m pip install --upgrade pip
+    pip-compile requirements.in
+    pip install -r requirements.txt
+    pre-commit install
+deps =
+    {[testenv:tests]deps}
+    {[testenv:linting]deps}
+    {[testenv:coverage]deps}
+    black
+    isort
+    pip-tools
+    pre-commit
+
+
 [testenv:docs]
-deps = -r requirements/docs.txt
+deps =
+    -rrequirements.txt
+    Sphinx
```

Comment:
ist das mit upper case `S`?

Reply:
in der Tat: https://pypi.org/project/Sphinx/ → `pip install Sphinx`